node/node.go

```diff
@@ -1129,6 +1135,38 @@ func (n *Node) DecodeAddressPubkey(pk string) ([]byte, error) {
 	return n.addressPubkeyConverter.Decode(pk)
 }
 
+// AddQueryHandler adds a query handler in cache
+func (n *Node) AddQueryHandler(name string, handler debug.QueryHandler) error {
+	if check.IfNil(handler) {
```

Comment:
Should also check if len(name) != 0

Reply:
Empty string is a valid identifier if we decide for that (I doubt about it, but it is possible). Added check.